src/dmd/expression.h

```diff
@@ -129,6 +129,7 @@ class Expression : public RootObject
     TOK op;                     // to minimize use of dynamic_cast
     unsigned char size;         // # of bytes in Expression so we can copy() it
     unsigned char parens;       // if this is a parenthesized expression
+    const(char)* stringified;  // for function calls (in case of __ARG_NAME__)
```

Comment:
Same here, the code implements `__ARG_STRING__`.

Reply:
done